cylc/flow/cycling/loader.py

```diff
@@ -63,8 +63,16 @@ class DefaultCycler:
     TYPE: str
 
 
+@overload
+def get_point(value: str, cycling_type: Optional[str] = None) -> PointBase: ...
```

Comment:
Can we do this, to avoid the style violation?:
```suggestion
def get_point(value: str, cycling_type: Optional[str] = None) -> PointBase:
    ...
```